sda_orchestrator/utils/consumer.py

```diff
@@ -37,12 +37,12 @@ def __init__(
         self.vhost = vhost
         self.max_retries = max_retries
         self.connection = None
-        self.ssl = bool(strtobool(os.environ.get("BROKER_SSL", "True")))
+        self.ssl = bool(strtobool(environ.get("BROKER_SSL", "True")))
         context = ssl.SSLContext(protocol=ssl.PROTOCOL_TLS)
         context.check_hostname = False
-        cacertfile = Path(f"{os.environ.get('SSL_CACERT', '/tls/certs/ca.crt')}")
-        certfile = Path(f"{os.environ.get('SSL_CLIENTCERT', '/tls/certs/orch.crt')}")
-        keyfile = Path(f"{os.environ.get('SSL_CLIENTKEY', '/tls/certs/orch.key')}")
+        cacertfile = Path(f"{environ.get('SSL_CACERT', '/tls/certs/ca.crt')}")
+        certfile = Path(f"{environ.get('SSL_CLIENTCERT', '/tls/certs/orch.crt')}")
+        keyfile = Path(f"{environ.get('SSL_CLIENTKEY', '/tls/certs/orch.key')}")
```

Comment:
```suggestion
        cacertfile = Path(environ.get('SSL_CACERT', '/tls/certs/ca.crt'))
        certfile = Path(environ.get('SSL_CLIENTCERT', '/tls/certs/orch.crt'))
        keyfile = Path(environ.get('SSL_CLIENTKEY', '/tls/certs/orch.key'))
```
Remove unnecessary f-strings